pandas/tests/groupby/test_groupby.py

```diff
@@ -2057,3 +2057,15 @@ def test_groups_repr_truncates(max_seq_items, expected):
 
         result = df.groupby(np.array(df.a)).groups.__repr__()
         assert result == expected
+
+
+def test_groupby_period_index():
+    periods = 2
+    index = period_range(start='2018-01', periods=periods, freq='M')
```

Comment:
Looks like you haven't run `black`, and perhaps not `flake8` either. See [code standards](https://pandas.pydata.org/docs/development/contributing.html#code-standards)